### ISSUE TITLE
Sometimes there's unwanted gap when drawing circle with line style

### ISSUE THREAD

**Issue by oliverzy** [pixijs/pixi.js#6327]
### Expected Behavior
[See Demo](https://pixi-playground.now.sh/loading.html)
There's no gap when drawing circle with linestyle
### Current Behavior
![image](https://user-images.githubusercontent.com/1321817/72331202-c008a900-36f2-11ea-8574-c42b3eb78070.png)
Sometimes there's small gap when drawing circle with linestyle, please see the screenshot above.

### Possible Solution
I don't know

### Steps to Reproduce
```javascript
import * as PIXI from 'pixi.js';
import { hslToRgb } from '../util';
import './main.scss';

class Particle {
  constructor(container, radius, radians, screenWidth, screenHeight, index) {
    this.g = new PIXI.Graphics();
    container.addChild(this.g);
    this.cx = radius*Math.cos(radians) + screenWidth/2;
    this.cy = radius*Math.sin(radians) + screenHeight/2;
    this.cr = index*2;
  }

  update() {
    this.cr += 0.1;
    if (this.cr > 40)
      this.cr = 0;
    this.alpha = Math.max(0, 1 - this.cr/20);
  }

  render() {
    this.g.clear();
    const color = PIXI.utils.rgb2hex(hslToRgb(200, 1, this.alpha));
    this.g.lineStyle(3, color, this.alpha);
    //this.g.beginFill(0xFF0000, this.alpha);
    this.g.drawCircle(this.cx, this.cy, this.cr*2);
    //this.g.endFill();
  }
}

function makeLoading(app) {
  const graphics = new PIXI.Graphics();
  let isPause = false;
  graphics.interactive = true;
  graphics.buttonMode = true;
  graphics.on('pointertap', () => {
    if (isPause)
      resume();
    else
      pause();
  });
  const container = new PIXI.Container();
  const particles = [];

  function play() {
    app.stage.addChild(graphics);
    app.stage.addChild(container);
    app.ticker.add(update);

    const radius = 100;
    const radians = 18*Math.PI/180;
    for (let i=0;i<20;++i) {
      const particle = new Particle(container, radius, radians*i, app.screen.width, app.screen.height, i);
      particles.push(particle);
    }
  }

  function pause() {
    isPause = true;
    app.ticker.remove(update);
  }

  function resume() {
    isPause = false;
    app.ticker.add(update);
  }

  function update() {
    graphics.clear();
    graphics.beginFill(0xF7F8FC);
    graphics.drawRect(0, 0, app.screen.width, app.screen.height);
    graphics.endFill();
    for (let particle of particles) {
      particle.update();
      particle.render();
    }
  }

  return {
    play
  }
}

function main() {
  const app = new PIXI.Application({
    width: 600, height: 600, backgroundColor: 0, resolution: window.devicePixelRatio || 1,
    view: document.getElementById('ground')
  });
  const loading = makeLoading(app);
  loading.play();
}

main();
```
### Environment

- **`pixi.js` version**: 5.2.0
- **Browser & Version**: Chrome 78
- **OS & Version**: MacOS 10.14.6
- **Running Example**: https://pixi-playground.now.sh/loading.html

**Comment by ivanpopelyshev** [comment]
Lets dig how that happened , and who made that: https://github.com/pixijs/pixi.js/blob/e0320c08763cf10a0b92c84bc8d69e412806e2a9/packages/graphics/src/utils/buildCircle.js#L48

**Comment by bigtimebuddy** [comment]
Fixed! Will be in the upcoming v5.2.1 release. Thanks for pointing out the bug @oliverzy

**Comment by oliverzy** [comment]
Thanks for all.
